ws_fnc/AI/fn_createGarrison.sqf

```diff
@@ -160,21 +176,20 @@ for "_x" from 1 to _int do {
 	_bpl deleteAt _i;
 
 	// Create a unit and move it into place
-  	_u = _grp createUnit [selectRandom _classes,_area,[],5,"NONE"];
+	_u = _grp createUnit [selectRandom _classes,_bp,[],5,"NONE"];
 	_u setPosATL _bp;
-  	dostop _u;
+	[_u] joinsilent _grp; //otherwise the side might be wrong
+	dostop _u;
+	//_u disableAI "PATH";
```

Comment:
Maybe a good idea to add the ASR_AI solution too?

Reply:
@ninjaProtocol 
I'm not aware of **`the`**` ASR_AI solution`.
But if you point me towards it, I can see what can be done. (Not sure it'll be part of this PR though)